### PR TITLE
Make FacadeAct.runRevenueOverview() work when toStart is empty

### PR DESCRIPTION
**Pull request by tbrent**
Not ready to merge; needs deploy

### CODE REVIEW

File: test/Facade.test.ts [hunk 1]
```diff
@@ -1164,6 +1164,18 @@ describe('FacadeRead + FacadeAct contracts', () => {
...
       )
+
+      // Nothing should be settleable
+      expect((await facade.auctionsSettleable(rsrTrader.address)).length).to.equal(0)
+
+      // Advance time till auction ended
+      await advanceBlocks(1 + auctionLength / 12)
+
+      // Settleable now
+      expect((await facade.auctionsSettleable(rsrTrader.address)).length).to.equal(0)
```
Comment (julianmrodri) [inline test/Facade.test.ts:1175]: Based on the comment I would expect a different thing here, seems its still not settleable?

Comment (tbrent) [reply test/Facade.test.ts:1175]: Very good question, investigating...

Comment (tbrent) [reply test/Facade.test.ts:1175]: Mmmm off-by-one-block read vs write difference. Advancing one further block solves the problem.

**Review by julianmrodri — APPROVED** [review]
LGTM

**Pull request merged**